### DISCUSSION TITLE
Window focus

### DISCUSSION THREAD

**Discussion by sheldon-walters** [nicolaasburgers/definition-library-releases#18] [Ideas]
When in the publish window - it would be great if the focus can be switched back to the main canvas ; so one can move, move and drill into clusters to interrogate them. This would help in coming up with meaningful names/tags in the publish window.

**Comment by nicolaasburgers** [comment]
If this change were made:
- What would be the behaviour you expect with the Publish window if you switch the currently-active document on the canvas?  Should the pop-up window remain, referring to a document non currently visible on the canvas?

**Reply by sheldon-walters** [reply]
Unaware of the programmatic focus constraints -- but ideally the Publish window stays visible, but control focus moves to the canvas, so you can zoom and pan, and drill into clusters -- then return to the publish window when ready.
